.gitignore

```diff
@@ -4,6 +4,7 @@
 *.swp
 *.swo
 *.exe
+/.vscode/
```

Comment:
I used `dlv` while developing this PR. VSCode stores `dlv` configurations in `.vscode`. VSCode is free to use for both open source and commercial development.